src/main/resources/xslt/preservica2schemaorg.xsl

```diff
@@ -375,32 +362,32 @@
       <!-- Create boolean for surround-->
       <xsl:choose>
         <xsl:when test="/xip:DeliverableUnit/Metadata/pbc:PBCoreDescriptionDocument/pbcoreInstantiation/formatChannelConfiguration = 'surround'">
-          <f:string key="kb:surround_sound"><xsl:value-of select="true()"/></f:string>
+          <f:boolean key="kb:surround_sound"><xsl:value-of select="true()"/></f:boolean>
         </xsl:when>
         <xsl:when test="/xip:DeliverableUnit/Metadata/pbc:PBCoreDescriptionDocument/pbcoreInstantiation/formatChannelConfiguration = 'ikke surround'">
-          <f:string key="kb:surround_sound"><xsl:value-of select="false()"/></f:string>
+          <f:boolean key="kb:surround_sound"><xsl:value-of select="false()"/></f:boolean>
         </xsl:when>
       </xsl:choose>
       <!-- Create boolean for color-->
       <xsl:choose>
         <xsl:when test="/xip:DeliverableUnit/Metadata/pbc:PBCoreDescriptionDocument/pbcoreInstantiation/formatColors = 'farve'">
-          <f:string key="kb:color">true</f:string>
+          <f:boolean key="kb:color">true</f:boolean>
```

Comment:
I don't understand why `<xsl:value-of select="false()"/>` is used in some places and `true` in other places!?

Reply:
Sloppiness from my side. I'll convert to using <xsl:value-of select="bool()"/> everywhere.